src/quo2/screens/main.cljs

```diff
@@ -23,6 +24,9 @@
               {:name      :quo2-tabs
                :insets    {:top false}
                :component tabs/preview-tabs}
+              {:name      :quo2-token_tag
```

Comment:
better to use `quo2-token-tag`, so that all screen names looks similar

Reply:
done